README.md

```diff
@@ -164,99 +188,71 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
 
 ### Updating Cumulus version and publishing to NPM
 
-All packages on master branch are automatically published to NPM.
-
-Follow the following steps to publish to NPM:
-
-#### 1. Create the release branch
-
-Create a new branch from `master` (make sure you have latest) and call it `release-version-<version_number>`. 
+#### 1. Create a branch for the new release
 
-Release PRs **MUST** be named with `release-` prefix. This will kick off the AWS integration tests in the CI process and ensures that package updates are fully tested on AWS before publication to NPM.
+The name is not important, but `release-x.y.z` seems like a good choice.
 
-#### 2. Update the Cumulus Version number
+#### 2. Update the Cumulus version number
 
-When changes are ready to be released, the Cumulus version number must be updated using semantic versioning.
+When changes are ready to be released, the Cumulus version number must be
+updated.
 
-Lerna handles the process of deciding which version number should be used as long as the developer decides whether the change is a patch or a minor/major change.
+Lerna handles the process of deciding which version number should be used as
+long as the developer specifies whether the change is a major, minor, or patch
+change.
 
 To update cumulus' version number run:
 
      $ yarn update
 
-You will be prompted to select the type of change (patch/minor/major). Lerna will update the version of all packages after the selection.
```

Comment:
Why did you remove this?

Reply:
It's redundant between the comment about "Lerna handles the process of deciding which version number should be used as long as the developer specifies whether the change is a major, minor, or patch change" and the screenshot.